packages/table/src/Row/InternalRowWithRT.tsx

```diff
@@ -87,7 +87,8 @@ const InternalRowWithRT = <T extends LGRowData>({
         className={cx(
           {
             [grayZebraRowStyles[theme]]: isOddVSRow && shouldAlternateRowColor,
-            [zebraStyles[theme]]: !virtualRow && shouldAlternateRowColor,
+            [zebraStyles[theme]]:
+              !virtualRow && shouldAlternateRowColor && !isSelected,
```

Comment:
@spark33 Should we be applying this condition to the above styles too? (`grayZebraRowStyles`)